### PR TITLE
Fix npm start for Windows

### PR DESCRIPTION
**Pull request by tisoap**
Command `npm start` didn't work on Windows because it could not find `examples/react-web` path. This was fixed by using two cd commands and avoiding using slashes. 

Fixes issue #451

#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] Test coverage is 100% (or you have a story for why it's ok).

### CODE REVIEW

**Comment by tisoap** [comment]
@googlebot I signed it!

**Comment by nicolodavis** [comment]
Thanks for the fix!

There is some additional discussion on https://github.com/nicolodavis/boardgame.io/issues/451 that suggests a different implementation.  Can you take a look?

**Comment by tisoap** [comment]
Following the discussion at #451 , I've updated my approach to replace `dev.sh` with a Node script, so it should now work on Windows.

File: scripts/install-web.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  console.log('Installing web dependencies...');
+  const npmInstall = spawn('npm', ['install'], { cwd: webExamplePath });
+
+  npmInstall.stdout.on('data', function(data) {
+    console.log(data.toString());
+  });
+
+  npmInstall.stderr.on('data', function(data) {
+    console.error(data.toString());
+  });
```
Comment (nicolodavis) [inline scripts/install-web.js]: The `npm install` subprocess seems to be silent for me (i.e. I can't see any output from it so it just appears that the terminal is hanging while it is doing the install).  Perhaps we're not subscribing to the data stream correctly?

File: scripts/install-web.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable unicorn/no-process-exit */
```
Comment (nicolodavis) [inline scripts/install-web.js]: Can you add the copyright notice here like we have for the other JS files in the repo?

File: scripts/install-web.js (hunk 1 above)
Comment (delucis) [reply scripts/install-web.js]: I’m not 100% sure of best practices, but if the aim is just to let `npm install` run and display it’s output, I think the simplest is using the `stdio` option:

```js
const npmInstall = spawn(
  'npm',
  ['install'],
  {
    cwd: webExamplePath,
    stdio: 'inherit'
  }
); 
```

This outputs everything straight to the parent `stdout`.

Comment (delucis) [reply scripts/install-web.js]: Node docs: https://nodejs.org/dist/latest-v12.x/docs/api/child_process.html#child_process_options_stdio

Comment (tisoap) [reply scripts/install-web.js]: Thanks @delucis , using `inherit` solved the issue!

**Comment by tisoap** [comment]
Got a hold on a Windows machine again and updated the install script. On Windows the install command is slightly different, so I had to add a check for that.

**Pull request merged**